.github/CODEOWNERS

```diff
@@ -10,3 +10,4 @@ charts/datadog/templates/container-system-probe.yaml @DataDog/ebpf-platform @Dat
 charts/datadog/templates/system-probe-configmap.yaml @DataDog/ebpf-platform @DataDog/container-integrations
 charts/datadog/templates/system-probe-init.yaml      @DataDog/ebpf-platform @DataDog/container-integrations
 charts/synthetics-private-location/                  @Datadog/synthetics
+charts/observability-pipelines-worker                @spencergilbert @lukesteensen
```

Comment:
hi @spencergilbert
we can also define a team for you 2. it will be simpler after to maintains

Reply:
👍 whatever the preference is from your side, I imagine I'd do most of the changes here - but it would be good for the observability pipelines time to have ownership.

Perhaps the better way forward is adding @DataDog/observability-pipelines as the owner, and then adding me/Luke to that team?

Reply:
Updated to the obs-pipeline team.